files/id_shared

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Applying the patch locally shows, that the file permissions for the private key are `0644` which leads to:

```
$ ssh -i files/id_shared sles@10.17.1.0
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@         WARNING: UNPROTECTED PRIVATE KEY FILE!          @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
Permissions 0644 for 'files/id_shared' are too open.
It is required that your private key files are NOT accessible by others.
This private key will be ignored.
Load key "files/id_shared": bad permissions
Password: 
```

Can you change the permissions to 440 please?

Reply:
ah sh*t, I swear I fixed this...I probably lost it in the rebase, sorry about that

Reply:
Fixed maybe with changing the keys to use the defaul user ones + jenkins keys